elt-pipelines/facility_ops/ingest/computing/jira/jira.py

```
@@ -35,27 +36,36 @@ class AtlassianCredentials(BaseSettings):
 
 class Extract(BaseExtract[AtlassianCredentials]):
     config_cls = AtlassianCredentials
+    issueKeys: list[str] = []
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)